understanding/audio-only-live.html

```diff
@@ -55,121 +53,92 @@
                <div class="box-i">
                   <ul>
-                     <li><a href="#brief">In Brief</a></li>
-                     <li><a href="#intent">Intent</a></li>
-                     <li><a href="#benefits">Benefits</a></li>
-                     <li><a href="#examples">Examples</a></li>
-                     <li><a href="#resources">Related Resources</a></li>
-                     <li><a href="#techniques">Techniques</a></li>
-                     <li><a href="#key-terms">Key Terms</a></li>
+                     <li><a href="#brief">要約</a></li>
+                     <li><a href="#intent">意図</a></li>
+                     <li><a href="#benefits">利点</a></li>
+                     <li><a href="#examples">事例</a></li>
+                     <li><a href="#resources">関連リソース</a></li>
+                     <li><a href="#techniques">テクニック</a></li>
+                     <li><a href="#key-terms">重要な用語</a></li>
                   </ul>
                </div>
             </nav>
          </aside>
          <main id="main" class="standalone-resource__main">
 
-         <!-- 日本語訳されるまでの一時的な注記 -->
-         <div class="untranslated-note"><p>このWCAG 2.2 解説書の日本語訳は作業中となっています。WCAG 2.1 解説書の日本語訳をご利用いただけます。<br>[ <a href="https://waic.jp/translations/WCAG21/Understanding/">WCAG 2.1 解説書</a> ]</p></div>
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				SC 1.2.9:</span>Audio-only (Live) (Level AAA)
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				達成基準 1.2.9:</span>音声のみ (ライブ) (レベル AAA)
+
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
-                  <p>An <a href="#dfn-alternative-for-time-based-media">alternative for time-based media</a> that presents equivalent information for <a href="#dfn-live">live</a>
-                     <a href="#dfn-audio-only">audio-only</a> content is provided.
+                  <p><a href="#dfn-live">ライブ</a>の<a href="#dfn-audio-only">音声しか含まない</a>コンテンツに対して、それと同等の情報を提示する、<a href="#dfn-alternative-for-time-based-media">時間依存メディアの代替</a>コンテンツが提供されている。
                      
                   </p>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Live audio can be understood by more people.</dd>
+                  <dt>目標</dt>
+                  <dd>ライブの音声をより多くの人々が理解できるようにする。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Provide a text equivalent for live audio-only content.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>ライブの音声のみのコンテンツと同等のテキストを提供する。</dd>
```

Comment:
下では equivalentを「同等の内容」と補っているので、こちらもそろえますか。

```suggestion
                  <dd>ライブの音声のみのコンテンツに対して、テキストによる同等の内容を提供する。</dd>
```